source/features/image-alternatives.js

```diff
@@ -0,0 +1,25 @@
+import {h} from 'dom-chef';
+import {safeElementReady} from '../libs/utils';
+
+export default async () => {
+	const imgContainers = document.querySelectorAll('.AdaptiveMedia > div > div > div');
+
+	for (let i = 0; i < imgContainers.length; i++) {
+		// Exit if it already exists
+		if (document.querySelector('div.refined-twitter_alt')) {
+			return;
+		}
+
+        var imgs = imgContainers[i].getElementsByTagName('img');
+        for (let j = 0; j < imgs.length; j++) {
+			let imgAlt = imgs[j].getAttribute('alt');
+
+			if(imgAlt) {
```

Comment:
Use early return here instead.

Reply:
Are you sure? As I see it, this control avoids ironing on a container already decorated. We can do the test as soon as possible.

Reply:
If `imgAlt` is empty you can return and stop processing. Inside the `imgAlt` condition, you check if it's already decorated and use another early return. :)

Reply:
If it's already decorated, we would have detect it earlier… As every container is immutable, if we already have decorated the image inside a container, there is no need to decorate again, or test each image…

Reply:
(I may have missed something, I only express what I think is happening)

Reply:
Sure, but instead of putting everythin inside the if, use the early return to exit if no `alt` is available, like this:

```javascript
if(!imgAlt) {
	return
}
var altDiv = document.createElement('div');
altDiv.className = 'refined-twitter_alt';
altDiv.innerText = imgAlt,
imgs[j].parentNode.insertBefore(altDiv, imgs[j].nextSibling)
```